IdentityServer4.NHibernate/IdentityServer4.Contrib.NHibernate.nuspec

```diff
@@ -15,10 +15,10 @@
     <tags>OAuth2 OAuth2.0 OpenIDConnect Security Identity IdentityServer IdentityServer4 NHibernate</tags>
     <dependencies>
       <group targetFramework="netcoreapp3.1">
-        <dependency id="IdentityServer4" version="4.0.0" />
-        <dependency id="IdentityServer4.Storage" version="4.0.0" />
-        <dependency id="NHibernate" version="5.2.0" />
-        <dependency id="AutoMapper" version="9.0.0" />
+        <dependency id="IdentityServer4" version="4.0.4" />
+        <dependency id="IdentityServer4.Storage" version="4.0.4" />
+        <dependency id="NHibernate" version="5.3.2" />
+        <dependency id="AutoMapper" version="10.0.0" />
```

Comment:
I would keep these versions as they are.
Keeping the major versions as the minimum requirement, doesn't force users to update their references.
Also for AutoMapper, the version 9.0.0, works perfectly for what we have to do here, so I would keep the version as is.

Reply:
Hi Alberto, 

the problem is that the versions in the nuspec need to match the ones from the project files, otherwise there are nasty compiler errors. i had an issue open about it but could not link it to the pull request somehow and closed it.  the libraries in this project need libs with a certain version but only get the one from the dependencies here and there is a mismatch. maybe you can see the closed issue for the exact compiler error.  dont you get compiler error when you try to pull the package from nuget?

Greetings

Ivan

Reply:
The versions specified in the nuspec file are the _minimum_ versions required for the library to run.

https://docs.microsoft.com/en-us/nuget/concepts/package-versioning

For example, if the nuspec file states that AutoMapper version 9.0 is required, if you add the library to a project that already has AutoMapper version 10, everything works.
But if you modify the nuspec file in order to require AutoMapper version 10, if you add the library to a project that already uses 9.0, you will force an AutoMapper upgrade for the hosting project (an maybe it's not needed).
That's why I would keep the nuspec versions as they are.

Regards,
Alberto